src/warnet/bitcoin.py

```diff
@@ -89,8 +117,12 @@ def grep_logs(pattern: str, show_k8s_timestamps: bool, no_sort: bool):
             continue
 
         # Get logs from the specific container
-        command = f"kubectl logs {pod_name} -c {container_name} --timestamps"
-        logs = run_command(command)
+        logs = sclient.read_namespaced_pod_log(
+            name=pod_name,
+            namespace=get_default_namespace(),
+            container=container_name,
+            timestamps=True,
+        )
```

Comment:
can also use the `_log()` internal function I added to get the log

Reply:
(or pod_log())

Reply:
fixed up here: https://github.com/bitcoin-dev-project/warnet/pull/614/commits/778a3ebab87474086fe39565e579169f49af2465